openwisp_controller/config/models.py

```diff
@@ -137,6 +137,7 @@ class Template(ShareableOrgMixin, AbstractTemplate):
 
     class Meta(AbstractTemplate.Meta):
         abstract = False
+        unique_together = (('organization', 'name'))
```

Comment:
I think this can be simply `unique_together = ('organization', 'name')` (without the double parenthesis)

Reply:
I read that it's supposed to be a tuple of tuples. I'll try this nonetheless.

Reply:
it can be also a tuple of tuples in case more complex checks should be performed, but keep in mind that what you have written now is not a tuple of tuple, this:

```python
# the outer parentheses do not instantiate a tuple but will
# be evaluated as precedence operator in an expression, so this:
(('organization', 'name'))
# will just become:
('organization', 'name')

# if you wanted to have a real tuple of tuples,
# you should have written it like this:
(('organization', 'name'),)
```

Reply:
Understood, @nemesisdesign